docs/semgrep-app/scm.md

```diff
@@ -0,0 +1,64 @@
+---
+slug: scm 
+append_help_link: true
+title: Integrating Semgrep into source code management (SCM) tools
+description: "Integrate Semgrep into self-hosted and custom SCM tools such as GitHub Enterprise and GitLab Self Hosted."
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Integrating Semgrep into source code management (SCM) tools
+
+Semgrep App's Team tier supports repositories hosted on both **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. Create multiple integrations in different self-hosted or custom servers.
+
+## Prerequisites
+
+You need the following permissions to integrate Semgrep into GHE or GLSM servers:
+
+* Permission to create a personal access token (PAT) for the repository to scan Semgrep on. This PAT is used to enable Semgrep App to create pull request (PR) or merge request (MR) comments for findings that can be [autofixed](../notifications/#automatically-fix-your-findings-through-pull-or-merge-requests).
+* Permission to add CI/CD secrets into your [GitHub](https://docs.github.com/en/actions/security-guides/encrypted-secrets) or [GitLab](https://docs.gitlab.com/ee/ci/secrets/) environments.
+
+## Integrating Semgrep into GitHub Enterprise or GitLab Self Managed 
+
+Integrate Semgrep into these custom source code management (SCM) tools by following the steps below:
+
+1. Sign in to [Semgrep App](https://semgrep.dev/login).
+2. Click **[Settings](https://semgrep.dev/orgs/-/settings)** > **Create new parent organization**.
+
+<div class = "bordered">
+
+![Screenshot of settings for parent organization](../img/app-parent-org.png "Screenshot of settings for parent organization")
+
+</div>
+
+3. Enter the name of your parent organization.
+4. Sign out, then sign in again to enable SCM management.
+5. Click **Settings** > **SCM management** > **Configure new SCM**.
+
+<div class = "bordered">
+
+![Screenshot of SCM configuration tab](../img/app-scm.png)
+
+</div>

```

Comment:
@s-santillan just not sure here about the borders. I generally think that a screenshot or anything in a numbered list should be intended with. I tried to test the difference (the screenshot above is intended and without border div).
<img width="990" alt="Screenshot 2022-05-17 at 13 16 37" src="https://user-images.githubusercontent.com/26669307/168799202-c2e3f2f9-644e-444b-a40e-7b675989659d.png">
<img width="986" alt="Screenshot 2022-05-17 at 13 16 23" src="https://user-images.githubusercontent.com/26669307/168799188-bdd5e3e0-7ecd-48de-9300-bd94f6884d1e.png">
:

Reply:
The problem is when the theme is white, it's hard to tell when the img starts and ends. Compare the two: 

![2022-05-17 19_59](https://user-images.githubusercontent.com/20766840/168805905-e9438b41-7fcb-4633-98ca-e469c82d0fc0.png)
.

Reply:
I see, let us keep borders then! @s-santillan

Reply:
And let's file a Docusaurus bug later... 

For the sake of future bug report:

The following code is displayed correctly in VS Code preview:

```md
6. Select your SCM provider and follow its procedure below:
<ol type="a">
    <li>For <b>GitHub Enterprise</b>, follow the steps below:</li>
        <ol>
        <li>Create a personal access token (PAT) by following the steps outlined in this  <a href="https://docs.github.com/en/enterprise-server@3.1/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token" target="_blank">guide to creating a PAT</a>.</li>
        <li>Enter the personal access token generated into the Access Token field.</li>
        <li>Enter your GHE base URL into the SCM Base URL.</li>
    </ol>
<li>For <b>GitLab Self Managed</b>, follow the steps below:</li>
    <ol>
        <li>Create a personal access token (PAT) by following the steps outlined in this <a href="https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html" target="_blank">guide to creating a PAT</a>.</li>
        <li>Enter the personal access token generated into the Access Token field.</li>
        <li>Enter your GLSM base URL into the <b>SCM Base URL</b> field.</li>
    </ol>
</ol>

7. Ensure that your SCM integration successfully detects repositories by following the steps below:
    1. Click **Projects > Setup New Project**.
    2. Select your CI provider.
    3. Semgrep App detects repositories from your SCM integration.
```

<img width="657" alt="Screenshot 2022-05-17 at 14 56 22" src="https://user-images.githubusercontent.com/26669307/168815841-8239fce9-3326-4303-8a02-dc8b0b4b4fb0.png">

But the result when Docusaurus built is the following:
<img width="827" alt="Screenshot 2022-05-17 at 14 57 27" src="https://user-images.githubusercontent.com/26669307/168816168-be729977-5a28-4aaf-8f98-65402509fb15.png">